src/components/ReactPlayer/ReactPlayer.tsx

```diff
@@ -391,6 +391,7 @@ export const ReactPlayerBlock = React.forwardRef<ReactPlayerBlockHandler, ReactP
                             onProgress={onProgress}
                             onEnded={onEnded}
                             aria-label={ariaLabel}
+                            previewTabIndex={-1}
```

Comment:
What does it mean?

Reply:
It means that preview image will not be focusable with Tab. At the same time we have on preview image play button, which is also accessible via Tab. I thought it would be sufficient to have it focusable